src/components/common/Sidebar/SidebarComponents.js

```diff
@@ -21,11 +19,10 @@ const MentorComponents = {
   1: <CalendarFeature />,
   2: <div>"Schedule Meeting"</div>,
   3: <div>"My Mentees" Component goes here</div>,
-  4: <div></div>,
-  5: <div>"Create Assignments" Component goes here</div>,
-  6: <div>"Request Resources" Component goes here</div>,
-  7: <div>"Track Resources" Component goes here</div>,
-  8: <RenderUpdateProfile />,
+  4: <div>"Request Resources" Component goes here</div>,
```

Comment:
Can this be turned into a single component like "Resources" which would then include two separate links with the option to request or track a resource? It would eliminate the need to create additional components while also allowing a user to do either or but from a  single page.

Reply:
What are the team's thoughts here? :)